pkg/virt-controller/services/template.go

```diff
@@ -70,6 +71,48 @@ func IsEmulationAllowed(store cache.Store) (bool, error) {
 	return useEmulation, nil
 }
 
+var GetNamespaceLimitRangesList = func(namespace string) (*k8sv1.LimitRangeList, error) {
```

Comment:
We should avoid rest-calls inside controller loops. Especially, if they would be done on every vmi creation. Can you use an informer instead and fetch the values from a cache?